lib/perlfaq4.pod

```diff
@@ -1116,6 +1116,43 @@ signal that I missed something:
 
     print $string;
 
+
+=head2 Does Perl have anything like Ruby's #{} or Python's f string?
+
+Unlike the others, Perl allows you to embed a variable naked in a quoted
+string C<"variable $variable">, when there isn't whitespace around the
+variable name you can add braces like C<"foo ${foo}bar">.
+
+An array can also be embedded directly in a string, and will be expanded
+by default with spaces between the elements. The default expansion can be
+changed by assigning a different string to the special variable B<$">,
+such as C<$" = ', ';>.
+
+Perl also supports references within a string providing the equivalent of
+the features in the other two languages.
+
+C<${\ ... }> embedded within a string will work for most simple statements
+such as an object->method call. More complex code can be wrapped in a do
+block C<${\ do{...} }>.
+
+When you want a list to be expanded per B<$">, use C<@{[ ... ]}>.
+
+    use Time::Piece; use Time::Seconds; my $t = Time::Piece->new;
+    say "Now is: ${\ $t->cdate() }";
+    say "Tomorrow: ${\ do{ my $T=Time::Piece->new + ONE_DAY ; $T->fullday }}";
+    my $scalar = 'STRING'; my @array = ( 'zorro', 'a', 1, 'B', 3 );
+    say "This is some scalar I have $scalar, this is an array @array.";
+    say "You can also write it like this ${scalar} @{array}.";
+    $" = ':';
```

Comment:
Do we want to encourage the use of `local` when changing special variables in example code?

Reply:
I was thinking about this. $" should either be over-ridden at the beginning of your program, or localized if it is  changed somewhere in the body. I again for brevity skipped mentioning that if you 'use English;' you can use the long name $LIST_SEPARATOR to make your code more readable.

Reply:
IMO it's a good idea to only recommend changing with with `local`.